### PR TITLE
Fix pretty printer output format

### PR DESCRIPTION
**Pull request by Garmelon**
It should now adhere to Blatt 4's specification (and do sensible things where the specification has gaps)

### CODE REVIEW

File: src/main/java/com/github/firmwehr/gentle/parser/ast/ClassDeclaration.java [hunk 1]
```diff
@@ -31,11 +34,34 @@ public ClassDeclaration withMainMethod(MainMethod mainMethod) {
...
+				if (it instanceof Method m) {
+					return m.name().ident();
+				} else if (it instanceof MainMethod m) {
+					return m.name().ident();
+				} else {
+					throw new IllegalArgumentException("expected Method or MainMethod");
+				}
+			})).collect(Collectors.toList());
+
+		p.add("public class ").add(name).add(" ");
```
Comment (pbrinkmeier) [inline src/main/java/com/github/firmwehr/gentle/parser/ast/ClassDeclaration.java]: `public class` is invalid.

**Review by pbrinkmeier — COMMENTED** [review]
Looks good to me. I will add another test then approve the PR.

**Review by pbrinkmeier — APPROVED** [review]
👍

File: src/main/java/com/github/firmwehr/gentle/cli/CommandArguments.java [hunk 2]
```diff
@@ -17,9 +17,12 @@
 	@Option(names = "--lextest", description = "Reads the given file and prints all tokens or aborts on first error")
 	boolean lextest();
 
-	@Option(names = "--parsetest", description = "Reads the given file and pretty-prints the parser AST")
+	@Option(names = "--parsetest", description = "Reads the given file and exits with 0 iff it is syntactically valid")
```
[the remark this answers is not in the document]
Comment (Garmelon) [reply src/main/java/com/github/firmwehr/gentle/cli/CommandArguments.java:20]: In that case, I'll leave it like that

**Pull request merged**